network/main.tf

```diff
@@ -0,0 +1,126 @@
+# --- network/main.tf ---
+
+data "aws_availability_zones" "available" {}
+
+resource "random_integer" "random" {
+    min = 1
+    max = 10
+
+}
+resource "random_shuffle" "az_list" {
+    input = data.aws_availability_zones.available.names
+    result_count = var.max_subnets
+}
+resource "aws_vpc" "atul_vpc" {
+    cidr_block = var.vpc_cidr
+    enable_dns_hostnames = true 
+    enable_dns_support = true
+
+    tags = {
+        Name = "atul_vpc-${random_integer.random.id}"
```

Comment:
Instead of a random integer, it's better to introduce a variable called 'env_code'
This will allow you to have more meaningful names. For example `atul_vpc-prod` or `atul_vpc-dev`

Reply:
Okay, thanks for it.